internal/commands/document.go

```diff
@@ -74,6 +78,7 @@ Set the URL where the policies are hosted at
 	}
 
 	cmd.Flags().StringP("output", "o", "policies.md", "Output location (including filename) for the policy documentation")
+	cmd.Flags().StringP("template-file", "", "", `File to read the template from (default: "")`)
```

Comment:
If no shorthand flag option is available, `String` should be used rather than `StringP`.